### ISSUE TITLE
Value for dropout?

### ISSUE THREAD

**Issue by cesarsouza** [deepmind/kinetics-i3d#8]
The provided i3d TensorFlow model specification includes dropout right before the last Unit3d layer. May I ask which value has been used for this dropout during the training on Kinetics and during the fine-tuning experiments?

I couldn't find it in the paper (but maybe its there and I am just blind).
